src/provider/model/Model.ts

```diff
@@ -66,7 +66,7 @@ export class Model {
     private readonly _onConnectionRegisteredInternal: Signal<[AppConnection]> = new Signal();
 
     constructor(
-        @inject(Inject.APP_DIRECTORY) directory: AppDirectory,
+        @inject(Inject.APP_DIRECTORY) directory: AppDirectory, // eslint-disable-line @typescript-eslint/indent
```

Comment:
All other places are doing this with `// eslint-disable-next-line`, would prefer consistency

Reply:
updated